templates/components/products/product-view.html

```diff
@@ -50,9 +50,11 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                     </span>
                 {{/if}}
                 {{#if settings.show_product_reviews}}
-                    <span class="productView-reviewLink">
+                    <span class="productView-reviewLink" role="button">
```

Comment:
Are there any limitations for using a button instead of span ?

Reply:
No limitations, have been changed, thank you